test/circuit_cutting/qpd/test_qpd.py

```diff
@@ -104,15 +104,13 @@ def test_decompose_qpd_instructions(self):
         with self.subTest("Empty circuit"):
             circ = QuantumCircuit()
             new_circ = decompose_qpd_instructions(QuantumCircuit(), [])
-            circ.add_register(ClassicalRegister(0, name="qpd_measurements"))
```

Comment:
These changes go with the earlier comment regarding adding empty QPD registers to subexperiments when no QPD measurements are present